simplecpp.cpp

```diff
@@ -1716,42 +1716,127 @@ namespace simplecpp {
         return ostr.str();
     }
 #else
-#define realFilename(f)  f
+    // Boring implementation, but could help in platform-independent tests:
+    std::string realFilename(const std::string &f)
+    {
+        return f;
+    }
 #endif
 
-    /**
-     * perform path simplifications for . and ..
-     */
-    std::string simplifyPath(std::string path)
+    static const char SEP='/';
+    static bool DOSDriveLetterPath(const std::string& path)
```

Comment:
as far as I see this is only used in the `DOSDriveLetterPathAbsolute`.. I would integrate this.